chain/network/src/raw/connection.rs

```diff
@@ -88,6 +166,51 @@ pub enum ConnectError {
     HandshakeFailure(HandshakeFailureReason),
     #[error("received unexpected message before the handshake: {0:?}")]
     UnexpectedFirstMessage(PeerMessage),
+    #[error(transparent)]
+    Other(anyhow::Error),
```

Comment:
ahh I added this here because `near_network::tcp::Stream::connect()` returns an `anyhow::Error`. But maybe it would be better to call it something else, yea